application/controllers/jwt.js

```diff
@@ -16,10 +17,23 @@ module.exports = {
       console.log('JWT Data is NOT valid:', decoded);
     callback(null, isValid);
   },
+
   createToken: (data) => {
     return jwt.sign(data, config.JWT.SERIAL, {
       algorithm: config.JWT.ALGORITHM,
       // expiresIn: 60 * 60 * 24 * 2 //two days
     });
-  }
+  },
+
+  createTokenForUser: (user) => {
```

Comment:
All available fields are now always included into the JWT, whereas before your changes only a few of them were included (e.g. "isReviewer"). If there is no data available to fill the field, it will end up with `isReviewer: undefined`. It would be better to check for emptiness and to fill it with `false` or to delete the field so it's not available at all (like before). Might also apply to "email".

Reply:
Only the fields `userid`, `userdata`, `email`, and `isReviewer` are ever included, not all available user attributes. You are right I should provide a `false` default value for `isReviewer`, but I assume every user has an `email`, no?

Reply:
A user should always have an email, that's right. But I don't know (I haven't read all the other code that you didn't touch) whether the object you're using (named `result`) will always contain an email.

Reply:
Looks good. Code is correct and preconditions are met.